main.cpp

```diff
@@ -52,85 +52,95 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: athlete
+//  action 1: an athlete trains in the gym.
+athlete.trains();
+//  action 2: an athlete competes in sporting events.
+athlete.competes();
+//  action 3: an  athlete maintains a healthy lifestyle such as good nutrition and rests.
+athlete.eatsHealthy();
```

Comment:
I'm marking this because:
1. your plain-english description is not an action, but is an ability or describes a state OR
2. the plain-english description is fine but the function name doesn't match OR
3. The way you wrote the name doesn't conform to the course coding standard:
    use `camelCase` for variable and function names.
    Type Names begin with a Capitalized first letter. All other symbols begin with a lower-case first letter.
- [ ] revise